corehq/apps/app_manager/static/app_manager/js/app_manager.js

```diff
@@ -540,8 +519,20 @@ hqDefine('app_manager/js/app_manager', [
 
     var _initNewModuleOptionClicks = function () {
         $('.new-module-option').on('click', function () {
-            var type = $(this).data('type');
-            console.log("You clicked module type:", type);
+            var moduleType = $(this).data('type');
+            $('#new-module-type').val(moduleType);
+            var $form = $('#new-module-form');
+            $('.new-module-icon').removeClass().addClass("fa fa-refresh fa-spin");
+            if (moduleType === "case") {
+                google.track.event("Added Case List Menu");
+                kissmetrix.track.event("Added Case List Menu");
+            } else if (moduleType === "survey") {
+                google.track.event("Added Surveys Menu");
+                kissmetrix.track.event("Added Surveys Menu");
+            }
```

Comment:
can this analytics code be added at the end so that any errors surrounding our analytics modules being blocked by ad blockers doesn't affect functionality

Reply:
This shouldn't be an issue, the tracking functions are set up to only run if the third-party scripts are loaded successfully: https://github.com/dimagi/commcare-hq/blob/891cc2068463782bb606b2699eced654627fca8f/corehq/apps/analytics/static/analytix/js/google.js#L84-L129